### ISSUE TITLE
Where/how are you using grid-editor?

### ISSUE THREAD

**Issue by Epskampie** [Frontwise/grid-editor#55]
As the creator of grid-editor, I'm interested in seeing in how it's used in the real world. Can you share a link or screenshot of your project, with a short description of the role grid-editor plays?

**Comment by oliverzwack** [comment]
hI, i'am using your superb tool for a small self-developed site management tool. 
All is perfect.. if I can make any suggestions.. a input field for a custom style-class like the ID field would be very helpful. We need in some case onetime-styling-classes for the row/column which we don't need to define in the js-script.
Thanks for your consideration ;-)
All the best from Munich!

**Comment by nestor-ps** [comment]
Hi, I am using the grid editor into an admin area for a tailor made ecommerce. I love this tool to create fast layouts. Very user friendly and clean output html. The ouput html is displayed in the frontend area as product description. 

![screenshoot](https://user-images.githubusercontent.com/1238026/30743484-1a6dd210-9f9e-11e7-8d01-e32c341f7b3e.PNG)

**Comment by maxomediaag** [comment]
We created a wordpress plugin which includes your grid editor and we enhanced it so we can use plugins (widgets) to use within your editor. See here:

![image](https://user-images.githubusercontent.com/12661103/32328971-e307cade-bfdb-11e7-95e6-8c830897795e.png)

![image](https://user-images.githubusercontent.com/12661103/32329005-fe35c61c-bfdb-11e7-9008-41fb861382df.png)

![image](https://user-images.githubusercontent.com/12661103/32329021-085cf372-bfdc-11e7-9773-afd8dba6d2cd.png)

The hole system is overriding the default wordpress wysiwyg-editor and the genereated code looks something like:

![image](https://user-images.githubusercontent.com/12661103/32329057-26ed3298-bfdc-11e7-9dcd-fa5f5a885f1e.png)

So basically if we have widgets which contains text it is stored as text in the editor-field and so also indexed by search or something else. No meta fields are needed :)

I know... there are some tradeoffs... for example you can't have something like "read more" which is often used by blogs. But we use this system to create normal websites and our clients can handle the whole responsive-thing a lot easier with this editor included :)

**Comment by rcraig12** [comment]
I have almost completed adding this to my laravel project. Although I have one issue that I just can't seem to figure out just yet. I'm storing the html that the grid editor creates back to my database. The idea is so that I can bring it all back on edit and continue making changes. However it seems to loose it's ability to fire the events for removing or adjusting the previously saved rows. Is there something special I need to do to get the gridEditor to start paying attention to the previously generated grids?

**Comment by rcraig12** [comment]
Solved it! Calling $('#Grid').gridEditor('gethtml'); makes the plugin parse the HTML tree and activate the previously edited markup. May be worth while getting the plugin to call that internally on init so that others don't get tripped up with that as an issue :)

**Comment by arikanorh** [comment]
We are using it in our dashboard like app. It Shows different datas on different type of graphics that is designed by end user.
